teal/crates/init.tl

```diff
@@ -80,6 +80,10 @@ function M.setup(cfg: Config)
 
     local group = vim.api.nvim_create_augroup("Crates", {})
     if state.cfg.autoload then
+        if vim.fn.fnamemodify(vim.api.nvim_buf_get_name(0), ":t") == "Cargo.toml" then
```

Comment:
To be consistent, since we use this already elsewhere 
```suggestion
        if vim.fn.expand("%:t") == "Cargo.toml" then
```

Reply:
Updated 👍🏼